README.md

```diff
@@ -7,7 +7,9 @@
 This repository is a collection of tools, objects and functions in *AssemblyScript*.
 
 - *Classes and helper functions* with **as-types**
-- *random draws* with **as-proba**
+
+- *Random draws* with **as-proba**
```

Comment:
Not just random draws ? Maybe just precise what his purpose is for SC developers ?

Reply:
What do you think about "*Most essential Classes and helper functions* with **as-types**" ?

Reply:
> What do you think about "_Most essential Classes and helper functions_ with **as-types**" ?

For the first comment, but I agree.

Reply:
What about: "*Probabilistic simulation functionalities for smart contracts*" ?